docs/graphkb/loading_data.md

```diff
@@ -82,6 +144,14 @@ Most popular resources which have pre-built loaders provided for GraphKB are lis
     Module used in other loaders for fetching publications (PubMed, PMC); genes (Entrez gene); RS ISs (snp), etc.
     from the NCBI Entrez API utitlies
 
+- ![logo](https://upload.wikimedia.org/wikipedia/commons/thumb/7/7d/Food_and_Drug_Administration_logo.svg/1280px-Food_and_Drug_Administration_logo.svg.png)
+
+    ### FDA Approval Announcements
+
+    [https://www.fda.gov/drugs/resources-information-approved-drugs/hematologyoncology-cancer-approvals-safety-notifications](https://www.fda.gov/drugs/resources-information-approved-drugs/hematologyoncology-cancer-approvals-safety-notifications)
+
+    Parses Oncology Approval Announcements from the FDA site, stores as evidence items
```

Comment:
ok, i'm not sure anymore whether to recommend ending all statements with periods or none, but probably best to do one or the other